### PR TITLE
fix: clients cores number setup

### PR DESCRIPTION
**Pull request by denise89p**

### CODE REVIEW

File: modules/clients/mount_wekafs.sh [hunk 1]
```diff
@@ -91,8 +94,8 @@ function retry {
 
 mount_command="mount -t wekafs -o net=udp $backend_ip/$FILESYSTEM_NAME $MOUNT_POINT"
 if [[ ${mount_clients_dpdk} == true ]]; then
-  getNetStrForDpdk $(($NICS_NUM-1)) $(($NICS_NUM)) "$gateways" "$subnets"
-  mount_command="mount -t wekafs $net -o num_cores=1 -o mgmt_ip=$eth0 $backend_ip/$FILESYSTEM_NAME $MOUNT_POINT"
+  getNetStrForDpdk 1 $NICS_NUM "$gateways" "$subnets" "-o net="
```
Comment (assafgi) [inline modules/clients/mount_wekafs.sh:97]: here you pass first gw then subnets and above function expects the other way

Comment (denise89p) [reply modules/clients/mount_wekafs.sh:97]: right we are not using subnets, I will remove it

File: modules/clients/main.tf [hunk 2]
```diff
@@ -10,18 +10,19 @@ data "azurerm_subnet" "subnet" {
...
     subnet_range    = data.azurerm_subnet.subnet.address_prefix
   })
 
   mount_wekafs_script = templatefile("${path.module}/mount_wekafs.sh", {
-    all_subnets        = split("/", data.azurerm_subnet.subnet.address_prefix)[0]
-    all_gateways       = cidrhost(data.azurerm_subnet.subnet.address_prefix, 1)
-    nics_num           = var.nics_numbers
-    backend_lb_ip      = var.backend_lb_ip
-    mount_clients_dpdk = var.mount_clients_dpdk
+    all_subnets                  = split("/", data.azurerm_subnet.subnet.address_prefix)[0]
```
Comment (assafgi) [inline modules/clients/main.tf:21]: can't all_subnets be removed?

File: modules/clients/mount_wekafs.sh [hunk 3]
```diff
@@ -38,20 +38,22 @@ weka local stop && weka local rm -f --all
 
 gateways="${all_gateways}"
 subnets="${all_subnets}"
```
Comment (assafgi) [inline modules/clients/mount_wekafs.sh:40]: can it be removed?

**Review by assafgi — APPROVED** [review]

**Pull request merged**